letsdrill.rb

```diff
@@ -1,24 +1,33 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if integer >= 90
+    return "A"
+  elsif integer >= 80 and integer < 90
+    return "B"
+  elsif integer >= 70 and integer < 80
+    return "C"
+  elsif integer >= 60 and integer < 70
+    return "D"
+  else
+    return "F"
+  end
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  array.min{|a,b| a.size <=> b.size }
```

Comment:
This is a great solution. Its about as good as you can write it in most languages.

Would you believe [Ruby has a method](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-min_by) that can simplify this even further?